src/icons/fileIcons.ts

```diff
@@ -1864,7 +1864,19 @@ export const fileIcons: FileIcons = {
     { name: 'advpl_include', fileExtensions: ['ch'] },
     { name: 'codeowners', fileNames: ['codeowners'] },
     { name: 'gcp', fileNames: ['.gcloudignore'] },
-    { name: 'disc', fileExtensions: ['iso'] },
+    { name: 'disc', 
+      fileExtensions: [
+        'iso',
+        'vdi',
+        'vhd',
+        'vmdk',
+        'hdd',
+        'qcow',
+        'qcow2',
+        'qed',
+        'dmg'
```

Comment:
Can you please insert a trailing comma here? 🙈
```suggestion
        'dmg',
```